ext/standard/basic_functions.c

```diff
@@ -2477,7 +2477,7 @@ PHP_FUNCTION(register_tick_function)
 
 	if (!zend_is_callable(&tick_fe.arguments[0], 0, &function_name)) {
```

Comment:
function_name here is no longer being used, so the argument can be removed. Though we could also just keep it in the error message.

Reply:
I readded it